src/main/java/pl/edukacyjni/user/UserService.java

```diff
@@ -9,15 +9,38 @@
 @Service
 public class UserService {
 
-    private static final List<User> USERS = new ArrayList<>(){{
+    private static final List<User> USERS = new ArrayList<>() {{
         add(new User(1L, "michaelPaello", "kochamPierogi"));
         add(new User(2L, "theGhostCasper", "huliganJezusaSt0"));
         add(new User(3L, "stNicolas", "haslo123!"));
         add(new User(4L, "santiago", "deadline"));
         add(new User(5L, "was-far", "abcd"));
     }};
 
-    public List<User> getAllUsers() {
-        return USERS;
+    private UserDTO convertUserToUserDTO(User user) {
```

Comment:
Please create a new class that would include that method.
Call it something like UserMapper and the method itself should be then called map or mapToDto